ci-operator/step-registry/rosa/terraform/e2e/teardown/rosa-terraform-e2e-teardown-ref.yaml

```diff
@@ -9,11 +9,12 @@ ref:
       memory: 300Mi
   timeout: 2h0m0s
   env:
-  - name: GATEWAY_URL
-    default: "https://api.stage.openshift.com"
   - name: TOKEN_URL
     default: "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token"
-  - name: OPENSHIFT_VERSION
```

Comment:
I think we should have a ``documentation`` entry here to explain that if the openshift-version is empty we're choosing the version based on the channel-group:
```
 - name: OPENSHIFT_VERSION
   default: ""
   documentation: ...
 - name: CHANNEL_GROUP
   default: "stable"
   documentation: ...
```

(see comment above about why separating the variables might be a more desired option)

Reply:
In this case of terraform variables. This design (variables in TF_VARS in terraform.tfvars format) is to make it dynamic (adding future variables, and moving directly to terraform, without dealing with each parameter in the wrapper code.

Reply:
> and moving directly to terraform, without dealing with each parameter in the wrapper code
yeah, but in turn we need to add the additional code of ``| grep 'url' | awk -F '=' '{print $2}' | sed 's/[ |"]//g')`` to extract the value from ``TF_VARS``. so it gets us to a place where we don't save code but just invest it on another place

> This design is to make it dynamic
it could be good if we only pass ``TF_VARS`` as-is to ``terraform apply``, but if we do process it during the step's code it means it isn't good for generalization. At least that's my opinion, @nirarg wdyt?